### ISSUE TITLE
TypeError: issubclass() arg 1 must be a class

### ISSUE THREAD

**Issue by TagnumElite** [sphinx-doc/sphinx#3767]
Subject: I tried to `make html` but it failed

### Problem
- issubclass(x, enum.Enum) Exception: arg 1 must be a class

#### Procedure to reproduce the problem
```
D:\GitHub\VectorBot\docs> make html
Running Sphinx v1.5.5
loading pickled environment... not yet created
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 5 source files that are out of date
updating environment: 5 added, 0 changed, 0 removed
reading sources... [100%] utils
Exception occurred:
  File "c:\users\user\appdata\local\programs\python\python36-32\lib\site-packages\sphinx\util\inspect.py", line 107, in isenumclass
    return issubclass(x, enum.Enum)
TypeError: issubclass() arg 1 must be a class
The full traceback has been saved in C:\Users\user\AppData\Local\Temp\sphinx-err-l8jae957.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

#### Error logs / results
[sphinx-err-l8jae957.txt](https://github.com/sphinx-doc/sphinx/files/1012353/sphinx-err-l8jae957.txt)

#### Expected results
Create an HTML version of the docs

### Reproducible project / your project
TagnumElite/VectorBot

### Environment info
- OS: Win
- Python version: 3.6
- Sphinx version: 1.5.5

**Comment by TagnumElite** [comment]
I know what went wrong and all but I'll just leave this here just in case you need it!

**Comment by TagnumElite** [comment]
Fixed in the new version. My Bad

**Comment by shimizukawa** [comment]
Yes, it was fixed by #3732 and has been shipped as sphinx-1.5.6.
Thanks for your reporting and fix confirmation!
